### DISCUSSION TITLE
How to use Gemini api in RAGflow

### DISCUSSION THREAD

**Discussion by Lycnkd** [infiniflow/ragflow#1796] [Q&A] [answered]
It is mentioned in doc of RAGflow that google Gemini api has been supported. However, I have checked it in Model Providers page from the running docker image and also the service_conf.yaml file and neither of them indicate the availability of Gemini api. So I just want to ask for a hand about how to use Gemini api in RAGflow.
![屏幕截图 2024-08-03 001713](https://github.com/user-attachments/assets/9390516f-efac-4515-a579-b21a495f7cc9)

**Comment by hangters** [comment, marked as answer]
hi @Lycnkd  do you update your ragflow ? if not ,update ragflow to the latest or dev version .
![屏幕截图_5-8-2024_114553_ragflow io](https://github.com/user-attachments/assets/86491419-6786-4cfb-8d3b-6b829b6ab090)

**Reply by Lycnkd** [reply]
Thanks for answering, it works! And I think gemini only appears in a RAGflow dev version.
